src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -96,7 +96,7 @@ private set
         internal BoundExpression MakeInvocationExpression(
             BinderFlags flags,
             SyntaxNode node,
-            BoundExpression? receiver,
+            BoundExpression receiver,
```

Comment:
`MakeInvocationExpression` below, which is called unconditionally, asserts that the receiver should not be null. We weren't using this incorrectly anywhere that I can see, but the current annotation is still a bug.